Perl/Decoder/t/091_thaw_order.t

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env perl
+use warnings;
```

Comment:
`use strict;`, please? :-)

(It was implied by `use v5.20`, but we can't use that trick with perls before v5.12.)

Reply:
Oh, yes. of course. Done.